faq.md

```diff
@@ -0,0 +1,195 @@
+  really matter if you specify, say, 1 or 2 or 11 am for a forecast that is
+  issued on the hour. But you could also do things like issue time of day =
+  0045, lead time to start = 75 minutes to represent the start of the CAISO real
+  time market window. The flexibility is powerful but a bit confusing.
+
+## Dashboard
+{: .anchor }
+---
+
+- Q. Can I instruct the Arbiter to create reference forecasts using NWP
+  data or persistence models for my own sites?
+
+  A. Not currently, but we hope to add this feature for historical data. You can
+  install
+  [`solarforecastarbiter-core`](https://solarforecastarbiter-core.readthedocs.io/en/latest/installation.html)
+  to run these models on your local machine and obtain true forecasts. We can
+  also run custom reference forecasts in a
+  [trial](/documentation/dashboard/trials/).
+
+
+## API
+{: .anchor }
+---
+
+- Q. Is there a limit to the number of requests per hour that I can make?
+
+  A. The Arbiter API does not currently impose a limit. For requests of
+  reasonable size (i.e. response time is much less than request frequency) you
+  can hit the server as often as you want. However, the Auth0 API does impose a
+  limit, so if you are obtaining a new token with every request then you may be
+  limited by Auth0.
+
+- Q. Should I obtain a new token with every request?
+
+  A. Generally you should only obtain a new token when your current token is
+  nearing or past its expiration. Obtaining a token more frequently than is
+  necessary runs the risk of hitting Auth0's rate limit.
+
+- Q. Can I upload data to multiple forecasts or observations at once?
+
+  A. No. A separate POST request must be made to each forecast or observation
+  endpoint. The benefit is that the format is exactly the same for all uploads,
+  so there is no need to worry about any nuance in file names, csv column names,
+  or json keys.
+
+- Q. How do I associate my data with UUIDs?
+
+  A. A brute force answer is a hardcoded dictionary or other mapping with keys
+  that are your e.g. PI names and values that are the SFA UUIDs. Another option
+  is to record your own resource name in the SFA’s observation metadata extra
+  parameters field, then search/filter the observations for the right one/ones,
+  then pull the UUID from the found object.
+
+- Q. Do you have any examples?
+
+  A. The [API documentation](https://api.solarforecastarbiter.org/) contains a
+  number of request and response examples. For examples using the Python API
+  wrapper, please see the [data_upload_download
+  notebook](https://github.com/SolarArbiter/workshop/blob/master/data_upload_download.ipynb)
+  in our [workshop repository](https://github.com/SolarArbiter/workshop). You
+  can actually run that notebook using
+  [MyBinder](https://mybinder.org/v2/gh/SolarArbiter/workshop/master) if you
+  want a very quick start.
+
+- Q. I can access the dashboard using my credentials but not the API. Why not?
```

Comment:
```suggestion
- Q. I'm having trouble connecting to the API or requesting an access token securely. What gives?
```
Maybe we should reference this older issue when this first cropped up? https://github.com/SolarArbiter/solarforecastarbiter-core/issues/474#issuecomment-638601890 The more recent issue was in requesting an access token from auth0, but it looks like it has cropped up before as well. I'll update the answer in another comment.